subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishDependenciesIntegTest.groovy

```diff
@@ -158,11 +158,15 @@ class MavenPublishDependenciesIntegTest extends AbstractIntegrationSpec {
 
         where:
         plugin         | gradleConfiguration  | mavenScope
-        'java'         | 'compile'            | 'runtime'
+        'java'         | 'compile'            | 'compile'
+        'java'         | 'runtime'            | 'compile'
```

Comment:
Why are we also changing runtime dependencies to 'compile' scoped maven dependencies?  Is that intentional?